docs/usecase.rst

```diff
@@ -37,14 +72,37 @@ compute/data/communications abstractions,
 quality of service guarantees,
 and performance optimizations.
 
+.. seealso:: :doc:`executor`
+
 Generic sequential task execution
 ---------------------------------
 
 Units of work (nodes) describe the inputs and expected outputs,
 and identify a module able to perform the operation.
 
+| 1. Receive a node of work to execute.
+| 2. Import a module implementing the operation.
+| 3. Initialize or restore checkpoint state.
+| 4. Prepare operation resources.
+|    3.1 Request or confirm locality of input data.
+|    3.2 Create data publishing resource for operation.
+|    3.3 Prepare compute environment for execution.
+|    3.4 Use operation module helpers to construct input objects.
+| 5. Provide resources and execute operation.
+| 6. Finalize checkpoint state.
+| 7. Note or confirm locality of operation results.
```

Comment:
This refers to the data representing the results of the operation?

Reply:
Yes, though an actual memory buffer or open filehandle may not exist. The result may be confirmation of the existence, validity, and location of a filesystem object (or database record, etc, local or remote) and sufficient metadata to locate and retrieve it.